core/src/main/scala/kafka/zk/ZkMigrationClient.scala

```diff
@@ -211,12 +214,38 @@ class ZkMigrationClient(zkClient: KafkaZkClient) extends MigrationClient with Lo
     }
   }
 
+  def migrateAcls(recordConsumer: Consumer[util.List[ApiMessageAndVersion]]): Unit = {
+    // This is probably fairly inefficient, but it preserves the semantics from AclAuthorizer (which is non-trivial)
+    var allAcls = new scala.collection.immutable.TreeMap[ResourcePattern, VersionedAcls]()(new ResourceOrdering)
+    def updateAcls(resourcePattern: ResourcePattern, versionedAcls: VersionedAcls): Unit = {
+      allAcls = allAcls.updated(resourcePattern, versionedAcls)
+    }
+
+    AclAuthorizer.loadAllAcls(zkClient, this, updateAcls)
```

Comment:
Yes, good call-out -- we will need documentation on this. We will only support migrating ACLs from AclAuthorizer to StandardAuthorizer. Otherwise, users can continue using whatever authorizer they had configured in ZK mode. In fact, AclAuthorizer _could_ still be used in KRaft mode, but we want to discourage that :)